<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWidget</class>
 <widget class="QWidget" name="LoginWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>251</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">LoginWidget</string>
  </property>
  <layout class="QVBoxLayout" name="pVerticalLayout_">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>13</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="pGroupBox_">
       <property name="title">
        <string notr="true">Login</string>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>&amp;Password:</string>
          </property>
          <property name="buddy">
           <cstring>pLineEditPassword_</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="pLineEditPassword_">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QCheckBox" name="pCheckBoxShowCharacters_">
          <property name="text">
           <string>&amp;Show Characters</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <spacer name="pVerticalSpacerPasswordHint_">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>13</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="pLabelPasswordHint_">
          <property name="text">
           <string notr="true">&lt;a href=fake_url&gt;Password Hint:&lt;/a&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="pLabelPasswordHintValue_">
          <property name="text">
           <string notr="true">hint line1
line2</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="pVerticalSpacerReadOnly_">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="pCheckBoxReadOnly_">
       <property name="text">
        <string>&amp;Read only</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>13</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="::cppdevtk::gui::CapsLockWidget" name="pCapsLockWidget_" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>::cppdevtk::gui::CapsLockWidget</class>
   <extends>QWidget</extends>
   <header location="global">cppdevtk/gui/caps_lock_widget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pLineEditPassword_</tabstop>
  <tabstop>pCheckBoxShowCharacters_</tabstop>
  <tabstop>pCheckBoxReadOnly_</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
